sys-auth/libyubikey/libyubikey-1.13.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Low level C SDK for the Yubico YubiKey authentication device"
+DESCRIPTION="Low-level C software development kit for the Yubico YubiKey \
+authentication device."
+HOMEPAGE="https://developers.yubico.com/yubico-c/"
+COPYRIGHT="2008-2015 Yubico AB"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://developers.yubico.com/yubico-c/Releases/libyubikey-$portVersion.tar.gz"
+CHECKSUM_SHA256="04edd0eb09cb665a05d808c58e1985f25bb7c5254d2849f36a0658ffc51c3401"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libyubikey$secondaryArchSuffix = $portVersion
+	cmd:modhex$secondaryArchSuffix = $portVersion
+	cmd:ykparse$secondaryArchSuffix = $portVersion
+	cmd:ykgenerate$secondaryArchSuffix = $portVersion
+	lib:libyubikey$secondaryArchSuffix = 0.1.7
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libusb$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libyubikey${secondaryArchSuffix}_devel = $portVersion
+	devel:libyubikey$secondaryArchSuffix = 0.1.7
+	"
+REQUIRES_devel="
+	libyubikey$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	lib:libcurl$secondaryArchSuffix
+	lib:libusb$secondaryArchSuffix
+	"
+
```

Comment:
Sorry, I had missed your message.
And I just realized I forgot to mention our HaikuPorter Guidelines!
https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering

BTW, if you would like to work on other recipes, we have a lot of these [Create a recipe for ...](https://codein.withgoogle.com/tasks/?sp-organization=5761416665497600&sp-search=Create%20a%20recipe%20for) tasks ;-)